pyproject.toml

```diff
@@ -33,7 +33,7 @@ classifiers = [
 ]
 requires-python = '>=3.9'
 dependencies = [
-    'audbackend[artifactory] >=2.0.0',
+    'audbackend[all] @ git+https://github.com/audeering/audbackend.git@main',
```

Comment:
Needs to be updated after the 2.1.0 release of `audbackend` is done (https://github.com/audeering/audbackend/pull/240).

Reply:
```suggestion
    'audbackend[all] >=2.1.0',
```